kedro_azureml/generator.py

```diff
@@ -191,12 +195,30 @@ def _construct_azure_command(
             environment=self._resolve_azure_environment(),  # TODO: check whether Environment exists
             inputs={
                 self._sanitize_param_name(name): (
-                    Input(type="string") if name in pipeline.inputs() else Input()
+                    Input(type="string")
+                    if name.startswith(PARAMS_PREFIX)
+                    or (
+                        name in pipeline.inputs()
```

Comment:
This one is pretty complex condition (also - it's used 2 times https://github.com/getindata/kedro-azureml/pull/60/files#diff-f3c50dbd7c6e2ab65abed2d289a431a0065eeefa1f717cc0b47c5182aba16494R329), please extract it to a separate method in the `AzureMLPipelineGenerator` class and give it a meaningful name (don't worry if it's overly verbose like `is_defined_and_not_folder_dataset`).